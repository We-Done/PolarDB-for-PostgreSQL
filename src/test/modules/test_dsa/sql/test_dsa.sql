CREATE EXTENSION test_dsa;

<<<<<<< HEAD
SELECT test_dsa_random(3, 5, 1024, 4096, 'random');
SELECT test_dsa_random(3, 5, 1024, 4096, 'forwards');
SELECT test_dsa_random(3, 5, 1024, 4096, 'backwards');

SELECT count(*) from test_dsa_random_parallel(3, 5, 1024, 8192, 'random', 5);
SELECT count(*) from test_dsa_random_parallel(3, 5, 1024, 8192, 'forwards', 5);
SELECT count(*) from test_dsa_random_parallel(3, 5, 1024, 8192, 'backwards', 5);

SELECT test_dsa_oom();
=======
SELECT test_dsa_basic();
SELECT test_dsa_resowners();
>>>>>>> c1ff2d8bc5be55e302731a16aaff563b7f03ed7c
