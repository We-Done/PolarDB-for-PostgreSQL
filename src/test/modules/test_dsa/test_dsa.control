<<<<<<< HEAD
# dsa_test extension
comment = 'Tests for DSA'
=======
comment = 'Test code for dynamic shared memory areas'
>>>>>>> c1ff2d8bc5be55e302731a16aaff563b7f03ed7c
default_version = '1.0'
module_pathname = '$libdir/test_dsa'
relocatable = true
