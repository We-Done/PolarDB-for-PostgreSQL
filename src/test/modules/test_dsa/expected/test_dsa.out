CREATE EXTENSION test_dsa;
<<<<<<< HEAD
SELECT test_dsa_random(3, 5, 1024, 4096, 'random');
 test_dsa_random 
-----------------
 
(1 row)

SELECT test_dsa_random(3, 5, 1024, 4096, 'forwards');
 test_dsa_random 
-----------------
 
(1 row)

SELECT test_dsa_random(3, 5, 1024, 4096, 'backwards');
 test_dsa_random 
-----------------
 
(1 row)

SELECT count(*) from test_dsa_random_parallel(3, 5, 1024, 8192, 'random', 5);
 count 
-------
     5
(1 row)

SELECT count(*) from test_dsa_random_parallel(3, 5, 1024, 8192, 'forwards', 5);
 count 
-------
     5
(1 row)

SELECT count(*) from test_dsa_random_parallel(3, 5, 1024, 8192, 'backwards', 5);
 count 
-------
     5
(1 row)

SELECT test_dsa_oom();
 test_dsa_oom 
--------------
=======
SELECT test_dsa_basic();
 test_dsa_basic 
----------------
 
(1 row)

SELECT test_dsa_resowners();
 test_dsa_resowners 
--------------------
>>>>>>> c1ff2d8bc5be55e302731a16aaff563b7f03ed7c
 
(1 row)

